python/lancedb/table.py

```diff
@@ -341,16 +341,18 @@ def checkout(self, version: int):
             raise ValueError(f"Invalid version {version}")
         self._reset_dataset(version=version)
 
-    def restore(self, version: int):
+    def restore(self, version: int = None):
         """Restore a version of the table. This is an in-place operation.
 
         This creates a new version where the data is equivalent to the
-        specified previous version. Note that this creates a new snapshot.
+        specified previous version. Data is not copied (as of python-v0.2.1).
```

Comment:
```suggestion
        specified previous version. Data is not copied (as of lancedb-v0.2.1).
```

Reply:
oh that's the anticipated tag name. We prefix it with `python-` to distinguish it from nodejs package releases.